frontend/src/app/signUp/SignUpApp.tsx

```diff
@@ -16,13 +16,15 @@ const SignUpApp = () => {
   );
   return (
     <Page isModalActive={modalActive}>
-      <Routes>
-        <Route path="/" element={<SignUpGoals />} />
-        <Route
-          path={"identity-verification"}
-          element={<Consent isModalActive={modalActive} />}
-        />
-      </Routes>
+      <main>
```

Comment:
(non-blocking, for another PR): it'd be great if we could move all these `main` tags out to the `Page` component instead of adding them for each separate component.

Reply:
Ticket here: https://github.com/CDCgov/prime-simplereport/issues/4287